### PR TITLE
Update installation data.

### PR DESCRIPTION
**Pull request by ahamed**
Pull Request for Issue #100.

### Summary of Changes
This PR updates the installation data and introduces a new dashboard module's arrangement system.

### Testing Instructions
Remove `configuration.php` file from the project root > Reinstall your Joomla! > Go to Dashboard > View the changes.

**Note:** Don't forget to run `npm run build:js && npm run build:css`

### Expected result
Screenshot 2019-11-28 at 1 35 21 PM

### Actual result
j4adminui-dashboard

### CODE REVIEW

File: administrator/components/com_modules/Model/ModuleModel.php [hunk 1]
```diff
@@ -1156,4 +1156,41 @@ protected function cleanCache($group = null, $client_id = 0)
 	{
 		parent::cleanCache('com_modules', $this->getClient());
 	}
+
+	/**
+	 * Save Dashboard module's position
+	 *
+	 * @param	array	$pks		Primary key of the modules.
+	 * @param	array	$position	Position array which indicates that the module on the left column or right column.
+	 * @return 	bool				Is the position is saved on the params or not.
```
Comment (brianteeman) [inline administrator/components/com_modules/Model/ModuleModel.php]:
```suggestion
	 * @return 	bool			If the position is saved on the params or not.
```

**Comment by kawshar** [comment]
Joomla 4 Dashboard
I can confirm the issue is fixed.

**Comment by joomlacorner** [comment]
1. Patch worked as Expected.  Is need to improve css for support screen resolution 1280x800?
2. Agreed with @brianteeman to changed text from "Is"  to 'If'

This my test result on 1280x800.
FireShot Capture 147 - Control Panel - Jo_ - http___localhost_8888_j4adminui_administrator_index php

**Comment by ahamed** [comment]
@joomlacorner could you please raise an issue for this responsive problem?

**Comment by davidnguyen-joomlashine** [comment]
I confirm the fix is correct. 
Screenshot

However there are some responsive issues. I will raise an issue for the response issue, for example 

Screenshot (1)

**Comment by kawshar** [comment]
I think the CI has a problem. Sometimes it works and sometimes doesn't.

**Pull request merged**